app/ui-sidenav/client/sideNav.js

```diff
@@ -8,6 +9,10 @@ import { SideNav, menu } from '../../ui-utils';
 import { settings } from '../../settings';
 import { roomTypes, getUserPreference } from '../../utils';
 import { Users } from '../../models';
+import { createTemplateForComponent } from '../../../client/reactAdapters';
+
+createTemplateForComponent('sidebarHeader', () => import('../../../client/components/sidebar/Header'));
+createTemplateForComponent('sidebarChats', () => import('../../../client/components/sidebar/Chats'), { renderContainerView: () => HTML.DIV({ style: 'display: flex; flex: 1 1 auto;' }) });// eslint-disable-line new-cap
```

Comment:
Why this is style is needed on the template's wrapper `div`? Could it be inside the component itself?